_pytest/config.py

```diff
@@ -1346,6 +1347,11 @@ def determine_setup(inifile, args, warnfunc=None):
                     is_fs_root = os.path.splitdrive(str(rootdir))[1] == '/'
                     if is_fs_root:
                         rootdir = ancestor
+    if rootdir_cmd_arg:
+        rootdir_abs_path = py.path.local(rootdir_cmd_arg)
```

Comment:
You should call `os.path.expandvars(rootdir_cmd_arg)` here, not in the test; we want to make sure the functionality is available to users after all. 😉